packages/grid/src/components/column/Column.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { ColumnProps } from './Column.types';
+
+import styles from '../../styles/column.module.css';
+
+const breakpoints = ['sm', 'md', 'lg', 'xl'];
+
+function Column({
+  element: Wrapper = 'div',
```

Comment:
by the naming convention of the props. let's use `Component` instead `element: Wrapper`